### PR TITLE
fix/gui-v2-fixed-issue-with-time-picker

### PR DESCRIPTION
**Pull request by mustafapc19**
Ref: [Comment](https://github.com/nocodb/nocodb/issues/2921#issuecomment-1201272932)
Fixes issue where `TimePicker` bugs out with mysql

### CODE REVIEW

File: packages/nc-gui-v2/components/cell/TimePicker.vue [hunk 1]
```diff
@@ -22,13 +22,20 @@ const localState = $computed({
...
+    if (!dateTime.isValid()) {
+      dateTime = dayjs(`1999-01-01 ${modelValue}`)
+    }
+    if (!dateTime.isValid()) {
       isTimeInvalid = true
       return undefined
     }
 
-    return dayjs(modelValue)
+    return dateTime
```
Comment (wingkwong) [inline packages/nc-gui-v2/components/cell/TimePicker.vue:38]: What about `dateTime.format('HH:mm:ss')` ?

**Comment by wingkwong** [comment]
Please rebase & run lint again

File: packages/nc-gui-v2/components/cell/TimePicker.vue (hunk 1 above)
Comment (mustafapc19) [reply packages/nc-gui-v2/components/cell/TimePicker.vue:38]: @wingkwong Ant date picker expects a dayjs object and takes a date format too. So this is taken care of.

Comment (wingkwong) [reply packages/nc-gui-v2/components/cell/TimePicker.vue:38]: Oh ok then.

**Review by wingkwong — APPROVED** [review]

**Pull request merged**